redis/ocsp.py

```diff
@@ -11,8 +11,50 @@
 from redis.exceptions import AuthorizationError, ConnectionError
 
 
+def _check_certificate(certificate):
```

Comment:
This method does not validate the signature on the stapled response. 
It should verify the signature using the responder's public key (the responder's certificate that includes its private key is included in the stapled OCSP response)